apps/tup-cms/docker-compose.dev.yml

```
@@ -47,8 +47,9 @@ services:
       - ./elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
       - tup_cms_es_data:/usr/share/elasticsearch/data
     container_name: tup_cms_elasticsearch
-    ports:
-      - 9201:9200
+    # To avoid exposing Elasticsearch to network (thus letting UT/ISO sniff port), do not set ports (per Jake R)
+    # ports:
+    #   - 9201:9200
```

Revert this, because #106.
```suggestion
    ports:
      - 9201:9200
```